processing/include/HpsEventFile.h

```diff
@@ -60,7 +60,7 @@ class HpsEventFile : public IEventFile {
         void close();
 
 
-    private:
+    private:        
```

Comment:
I don't think this whitespace change is needed.

Reply:
I promise these aren't intentional.